doc/guides/writing-and-running-benchmarks.md

```diff
@@ -62,8 +62,13 @@ from [nghttp2.org][] or built from source.
 
 ### Benchmark analysis requirements
 
-To analyze the results, `R` should be installed. Use one of the available
-package managers or download it from <https://www.r-project.org/>.
+To analyze the results statistically, [node-benchmark-compare][] must be used.
```

Comment:
Instead of using "must", why not keeping both? The user might have `R` already installed and they might prefer to not install `node-benchmark-compare`. The other way round is also true so I think keeping both options is the best solution.

Reply:
Sure. I'll try to find a wording that keeps both.

Reply:
Done. wdyt?